src/microbit/pxt_hjerneteppe/hjerneteppe.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "PXT: Hjerneteppe!"
+author: Helene Isnes
+language: nb
+---
+
+
+# Introduksjon {.intro} 
+
+"Hjernteppe!" er en huskelek hvor du skal huske stadig lengre rekker med bokstaver!
+
+I denne oppgaven skal vi bruke noe som kalles for en funksjon. En funksjon er et sett med flere kodelinjer (klosser) som til sammen utfører en oppgave. Å lage funksjoner er lurt dersom man skal gjøre oppgaven flere ganger underveis i programmet. Det kan også være lurt for å dele opp programmet i mindre deler.  
+
+
+# Steg 1: Velg fasit {.activity}
+
+*Det første vi skal kode er en funksjon som skal inneholde de forskjellige bokstavkombinasjonene man kan få som en oppgave. Funksjonen skal også ha med en variabel `Fasit` som skal velge hvilken oppgave som er den gjeldende.*
+
+## Sjekkliste {.check}
+
+- [ ] Lag en ny funksjon som du kaller `Velg fasit`. Du lager en ny funksjon med `Lag en funksjon...`-knappen i kategorien `Funksjoner` i `Avansert`. 
+
+- [ ] Sett en variabel kalt `Oppgaver` til en liste med tre eller flere tekstsstrenger med ulike variasjoner på rekkefølge på bokstavene A, B og C. Du kan få bruk for denne klossen: ![Bilde som viser klossen sett text list til](sett_liste_til.png). Koden settes inn i `Velg fasit`. 
+
+## {.tip}
+Forslag til tre ulike teksstrenger er: ABCCBABA, ACCBACBC og CBACCBAB
+##
+
+- [ ] En variabel `Fasit` skal settes til `Oppgaver` som får en verdi mellom 0 og antall oppgaver - 1. `Sett Fasit til`-klossen finner du i `Variabler`, ![Bilde av klossen list får en verdi](list_faar_verdi.png) finner du i `Lister` og `plukk et tilfeldig tall mellom...` finner du i `Matematikk`. 
+
+## OBS! {.tip}
+Hvis du endrer antall oppgaver i listen, må du også endre makstallet for det tilfeldige tallet du plukker. Makstallet skal være antall oppgaver - 1. 
```

Comment:
Kan vere lurt å merke antall oppgaver - 1 i fnuttar, for å vise betre at det er eit tal: `antall oppgaver - 1`